pymc3/tests/test_data_container.py

```diff
@@ -156,6 +157,26 @@ def test_shared_data_as_rv_input(self):
         np.testing.assert_allclose(np.array([2.0, 4.0, 6.0]), x.get_value(), atol=1e-1)
         np.testing.assert_allclose(np.array([2.0, 4.0, 6.0]), trace["y"].mean(0), atol=1e-1)
 
+    def test_shared_scalar_as_rv_input(self):
+        # See https://github.com/pymc-devs/pymc3/issues/3139
+        with pm.Model() as m:
+            shared_var = theano.shared(5.0)
+            v = pm.Normal("v", mu=shared_var, shape=1)
+
+        np.testing.assert_allclose(
+            v.logp({"v": 5.0}),
```

Comment:
My bad, you have to put the value in a list `v.logp({"v": [5.0]})`. The same for the one below.

Reply:
Ah, now I see why the test failed. I'll make the change.